src/scrapers/movie/IMDB.cpp

```diff
@@ -115,6 +116,15 @@ void IMDB::search(QString searchStr)
         new NetworkReplyWatcher(this, reply);
         connect(reply, &QNetworkReply::finished, this, &IMDB::onSearchIdFinished);
 
+    } else if (QVariant(includeAdult).toBool()) {
+        QUrl url = QUrl::fromEncoded(
+            QStringLiteral("https://www.imdb.com/search/title/?adult=include&view=simple&title=%1").arg(encodedSearch).toUtf8());
+        QNetworkRequest request(url);
+        request.setRawHeader("Accept-Language", "en"); // todo: add language dropdown in settings
```

Comment:
Btw: The results of IMDb are based on the users location and not this header. Setting this header is not necessary. That's something that I learned just recently... :-/